### ISSUE TITLE
no build for OS X?

### ISSUE THREAD

**Issue by mnemio** [huttonr/bootstrap4#5]
$ meteor update
This project is already at Meteor 1.2.1, the latest release.
=> Errors while upgrading packages:

While checking for huttonr:bootstrap4@4.0.0_7:
error: No compatible binary build found for this package. Contact the package
author and ask them to publish it for your platform.

OS X Version: 10.11.2 (15C50)
15.2.0 Darwin Kernel Version 15.2.0

**Comment by huttonr** [comment]
Odd, I could have sworn I published the binaries for OS X.

In any case this should now be resolved.  `meteor update` should work properly as of now.

Please let me know if it continues to misbehave.

**Edit: Having some trouble getting the build machine to not kick me off before it is built.  May just be a peak time but will continue to try.**

**Comment by mnemio** [comment]
Hmm, it still is not working. I get the same error with update. 
Next I tried removing the package and re-adding it. That did not work either, same error. 
I then re-added version 4.0.0_6 with the following results:

$ meteor add huttonr:bootstrap4@=4.0.0_6

Changes to your project's package version selections:

huttonr:bootstrap4         added, version 4.0.0_6
huttonr:bootstrap4-assets  added, version 4.0.0_7

**Edit: I just noticed your edit, which was not in the email I received. I will be patient.**

Thank you
